cranelift/codegen/meta/src/shared/instructions.rs

```diff
@@ -4220,6 +4220,28 @@ pub(crate) fn define(
         .constraints(vec![WiderOrEq(Float.clone(), FloatTo.clone())]),
     );
 
+    ig.push(
+        Inst::new(
+            "fvdemote",
+            r#"
+        Convert `x` to a smaller floating point format.
+
+        Each lane in `x` is converted to the destination floating point format
+        by rounding to nearest, ties to even.
+
+        Cranelift currently only supports two floating point formats
+        - `f32` and `f64`. This may change in the future.
+
+        Fvdemote differs from fdemote in that with fvdemote it targets vectors.
+        With Fvdemote there is no contraint on having the result type have the same
+        number of vector lanes as the input type.
```

Comment:
I would suggest trying to fit this under `fdemote`: the semantics are very close and we could document on `fdemote` that, when used on vector types, it's conversions are placed in the lowest lanes. E.g., when demoting lanes 0 and 1 of an `f64x2`, the results are placed in lanes 0 and 1 of the `f32x4`). We will also want to document that lanes 2 and 3 are zeroed out.